libs/juno-ui-components/README.md

```diff
@@ -17,8 +23,27 @@ in your app
 yarn install
 ```
 
+## Working With Tailwind
+Juno comes with [Tailwind](https://tailwindcss.com/) included, so when using Juno you automatically can use tailwind in your project.
+
```

Comment:
## Using the Juno Tailwind config
To be able to make full use of the predefined colors and other custom properties from the Juno UI components library you will need to include the ui components tailwind config into your application's tailwind config like this:

```js
module.exports = {
  presets: [
    require('juno-ui-components/tailwind.config')
  ],
 ...
}
```
Doing this lets you use Tailwind classnames from the Juno UI Components library.